### PR TITLE
Add entities

### PR DESCRIPTION
**Pull request by mick-roper**
* added entities to the `ref`

### CODE REVIEW

**Review by Lawouach — APPROVED** [review]
LGTM

File: docs/reference/entities/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## APIVersion
+
+Defines the APIVersion that this entity is compatible with.
+
+### Structure
+
+```yaml
+apiVerison: <string>
```
Comment (ciaranevans) [inline docs/reference/entities/index.md:29]:
```suggestion
apiVersion: <string>
```

File: docs/reference/entities/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## Kind
```
Comment (ciaranevans) [inline docs/reference/entities/index.md:32]: `kind:` is not in your Common Properties example

File: docs/reference/entities/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## Selectors
+
+Selectors are used compare elements of the `spec` of one manifest with the `labels` of another manifest.
+
+### Structure
+```yaml
+spec:
+    ***Selector:
```
Comment (ciaranevans) [inline docs/reference/entities/index.md:78]:
```suggestion
    <Kind>Selector:
```
?

File: docs/reference/entities/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    labels: {}
+    relatedTo: {}
+```
+
+### Properties
+
+| ref | description |
+|---|---|
+| labels | A dict<string,string> that describes the purpose of this manifest. Common properties include `name`, `owner`, `service`, `region`, etc.
+| relatedTo | A dict<string,string> that describes labels that this manifest relates to. For instance, `metadata.relatedTo.service: awesome-service` would form a relationship to all other manifests that have a `metadata.labels.service: awesome-service`. This relationship is can then be visualised in the ReliablyHQ apps and utilised by behaviour to invoke some functionality. |
```
Comment (ciaranevans) [inline docs/reference/entities/index.md:100]: Is this all a duplicate?

File: docs/reference/entities/indicator.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+## Properties
+| key | description |
+|---|---|
+| apiVersion | see [metadata](./index.md/#apiVersion) |
+| kind | see [metadata](./index.md/#kind) |
+| metadata | see [metadata](./inex.md/#metadata) |
+| spec | see [spec](./index.md/#spec) |
+| spec.from | The date that marks the start of the range that the indicator data relates to. |
```
Comment (ciaranevans) [inline docs/reference/entities/indicator.md:32]: Should we specify the accepted format for this?

File: docs/reference/entities/indicator.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+| spec.to | The date that marks the end of the range that the indicator data relates to. |
+| spec.percent | The percentage of the value defined by the indicaitor labels that relate to this period in time. |
```
Comment (ciaranevans) [inline docs/reference/entities/indicator.md:34]: Ditto on the allowed formats/types

File: docs/reference/entities/objective.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+## Properties
+| key | description |
+|---|---|
+| apiVersion | see [metadata](./index.md/#apiVersion) |
+| kind | see [metadata](./index.md/#kind) |
+| metadata | see [metadata](./inex.md/#metadata) |
+| spec | see [spec](./index.md/#spec) |
+| spec.indicatorSelector | see [selectors](./index.md/#selectors) |
+| spec.objectivePercent | the percent required by this objective |
+| spec.window | the window of time that should be used to select data that can be used to test the actual state of a system |
```
Comment (ciaranevans) [inline docs/reference/entities/objective.md:35]: Example/link to what a valid duration looks like?

File: docs/reference/entities/objectiveresult.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Structure
+
+```yaml
+apiVersion: reliablyhq.com/v1
+kind: ObjectiveResult
+metadata:
+    labels: {}
+    relatedTo: {}
+spec: {}
```
Comment (ciaranevans) [inline docs/reference/entities/objectiveresult.md:20]: Should this cover:

```
spec.indicatorSelector
spec.objectivePercent
spec.actualPercent
spec.remainingPercent
```
?

**Review by ciaranevans — CHANGES_REQUESTED** [review]

File: docs/reference/entities/index.md (hunk 1 above)
Comment (mick-roper) [reply docs/reference/entities/index.md:29]: done

File: docs/reference/entities/index.md (hunk 2 above)
Comment (mick-roper) [reply docs/reference/entities/index.md:32]: good catch - done

File: docs/reference/entities/index.md (hunk 3 above)
Comment (mick-roper) [reply docs/reference/entities/index.md:78]: Not necessarily always going to be the case....

File: docs/reference/entities/index.md (hunk 4 above)
Comment (mick-roper) [reply docs/reference/entities/index.md:100]: yep - shouldnt be there

File: docs/reference/entities/indicator.md (hunk 5 above)
Comment (mick-roper) [reply docs/reference/entities/indicator.md:32]: dunno - for now Im happy to leave it in and see what feedback we get

File: docs/reference/entities/indicator.md (hunk 6 above)
Comment (mick-roper) [reply docs/reference/entities/indicator.md:34]: same as above

File: docs/reference/entities/objective.md (hunk 7 above)
Comment (mick-roper) [reply docs/reference/entities/objective.md:35]: yup

File: docs/reference/entities/objectiveresult.md (hunk 8 above)
Comment (mick-roper) [reply docs/reference/entities/objectiveresult.md:20]: yep

**Pull request merged**